superset/assets/src/visualizations/index.js

```diff
@@ -108,7 +108,7 @@ const vizMap = {
   [VIZ_TYPES.word_cloud]: () =>
     loadVis(import(/* webpackChunkName: "word_cloud" */ './wordcloud/WordCloud.js')),
   [VIZ_TYPES.world_map]: () =>
-    loadVis(import(/* webpackChunkName: "world_map" */ './world_map.js')),
+    loadVis(import(/* webpackChunkName: "world_map" */ './WorldMap/adaptor.jsx')),
```

Comment:
Can we add `index.js` for each file and export `adaptor` there as default. Potentially we also want to just import the chart itself in the plugin system world.

Reply:
`index.js` file per vis folder could be a good idea +1